integration_tests/feature-store/server/Cargo.toml

```diff
@@ -11,17 +11,17 @@ edition = "2021"
 [dependencies]
 sqlx = { version = "0.7", features = ["runtime-tokio-native-tls", "postgres"] }
 tokio = { version = "1", features = ["full"] }
-tonic = "0.10.2"
+tonic = "0.11.0"
```

Comment:
Can we use the workspace version as well?

Reply:
It seems that things in `integration_tests` are separated from the main workspace.

Reply:
Didn't realize that.